AndroidIDPTemplate/app/src/com/salesforce/samples/salesforceandroididptemplateapp/MainApplication.kt

```diff
@@ -43,12 +44,30 @@ class MainApplication : Application() {
         MobileSyncSDKManager.initNative(applicationContext, MainActivity::class.java)
         MobileSyncSDKManager.getInstance().registerUsedAppFeature(FEATURE_APP_USES_KOTLIN)
 
-        /*
-         * Uncomment the following line to enable IDP login flow. This will allow the user to
-         * either authenticate using the current app or use a designated IDP app for login.
-         * Replace 'idpAppURIScheme' with the URI scheme of the IDP app meant to be used.
-         */
-        // MobileSyncSDKManager.getInstance().idpAppURIScheme = idpAppURIScheme
+        // Setting app as IDP for these allowed SP apps
+        MobileSyncSDKManager.getInstance().setAllowedSPApps(listOf(
```

Comment:
Now the IDP app needs to have more details about the SP apps it want to work with.